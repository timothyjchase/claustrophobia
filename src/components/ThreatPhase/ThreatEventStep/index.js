import React from 'react'
import PropTypes from 'prop-types'
import { Button } from 'semantic-ui-react'

const ThreatEventStep = ({ completeThreatEventStep }) => (
  <div>
    <Button.Group vertical fluid>
      <Button
        primary
        icon="play"
        content="Next"
        onClick={completeThreatEventStep}
      />
    </Button.Group>
  </div>
)

ThreatEventStep.propTypes = {
  completeThreatEventStep: PropTypes.func.isRequired,
}

export default ThreatEventStep
