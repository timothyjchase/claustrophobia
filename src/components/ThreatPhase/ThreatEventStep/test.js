import React from 'react'
import ThreatEventStep from './index'

describe('<ViewDocumentPage />', () => {
  describe('with document', () => {
    const props = {
      completeThreatEventStep: jest.fn(),
    }
    const component = shallow(<ThreatEventStep {...props} />)

    it('should match snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
