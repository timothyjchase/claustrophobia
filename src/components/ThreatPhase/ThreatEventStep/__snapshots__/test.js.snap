// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewDocumentPage /> with document should match snapshot 1`] = `
<div>
  <ButtonGroup
    fluid={true}
    vertical={true}
  >
    <Button
      as="button"
      content="Next"
      icon="play"
      onClick={[MockFunction]}
      primary={true}
      role="button"
    />
  </ButtonGroup>
</div>
`;
